_data/stage3.yml

```diff
@@ -218,29 +261,14 @@
                                 enumerable: false,
                                 configurable: true });
     }
-  presented:
-    - date: September&#xA0;2020
-      url: item
   has_specification: true
-  description: A proposal to add a <code>.item()</code> method to all the basic indexable classes (<code>Array</code>, <code>String</code>, <code>TypedArray</code>).
-  authors:
-    - Shu-yu Guo
-    - Tab Atkins
-  champions:
-    - Shu-yu Guo
-    - Tab Atkins
+  presented:
+    - date: "November\_2020"
+      url: at
```

Comment:
In [the row for `.at()` of the proposals repo's README](https://github.com/tc39/proposals/#stage-3), the cell for "November 2020" is supposed to be a link, but it seems to either have been omitted or will be coming later.

Previously, it linked to: https://github.com/tc39/notes/blob/master/meetings/2020-09/sept-22.md#item-for-stage-3

Having edge cases like these lead me to believe that we should implement some sort of data validation to catch such problems. Shouldn't be a problem, just wondering if this should be a cause to “stop the presses”.

Reply:
Those are never a link until the notes are published, so you'll always have to handle unlinked MonthYears.

Reply:
The website is still currently outdated because I was waiting for the notes to be published, and they still do not appear to be published. This is obviously going to be out of our control and I'm unsure how this will be preventable in the future.

This is obviously not a finished PR and I don't want it to seem that way. There are a few details that maintainers would want to verify are accurate. I will enumerate them momentarily. Please don't get the impression that I intend on abandoning this line of reasoning. The holiday weekend is probably to blame for late notes showing up.

Reply:
Today is the publish date, so I wouldn't say they're late until tomorrow - but yes, we'll always have to account for unpublished notes.

Reply:
I was not aware that there was an early or late part but sounds good. I'm still thinking about whether or not type checking or testing should be occurring in this PR (or even repo), but would like to see this landing here. There were basically no instructions on how to go about taking this issue on, would like it to wind up looking good in this repo.

Reply:
> The website is still currently outdated because I was waiting for the notes to be published, and they still do not appear to be published. This is obviously going to be out of our control and I'm unsure how this will be preventable in the future.

Taken care of in https://github.com/tc39/tc39.github.io/pull/237 (and adjustments to the algorithm + data type here).